docs/src/pages/components/text/examples/TextTheme.tsx

```diff
@@ -0,0 +1,48 @@
+import { AmplifyProvider, Flex, Text } from '@aws-amplify/ui-react';
+
+import { VARIATIONS_OPTIONS } from '../constants';
```

Comment:
Since the customer will not have access to the internal `constants` file, can we explicitly define the variation options here as an array? Ideally, a customer could copy and paste the example code and it would work out-of-the-box

Reply:
done